lib/tjbot.js

```diff
@@ -806,6 +806,8 @@ TJBot.prototype.read = function() {
  * Returns the photo data in a Buffer.
  */
 TJBot.prototype._captureImage = function(filePath) {
+    this._assertCapability('see');
```

Comment:
by current convention, we don't need to assert capabilities in private helper methods. end users should not be calling methods prefixed with a `_`, so there's no need to assert the capability here.